### PR TITLE
Check branch name

### PR DESCRIPTION
**Pull request by cole-c**
### Description
Created an empty test scene for Cole to mess around in and bullets now have a much shorter life expectancy than their vastly superior ancestors

### PR Checklist
- [x] New files committed (`git add .` to commit all new files)
- [x] New Prefabs work outside of your test scene (can throw into SampleScene to test)

### CODE REVIEW

**Comment by cole-c** [comment]
@warrenscipio @RyanCross plz review

File: Assets/Scripts/Projectile.cs [hunk 1]
```diff
@@ -5,6 +5,12 @@
 public class Projectile : MonoBehaviour
 {
     public GameObject hitEffect; //put some explosion or some shit here.
+    public float timeToLive = 2.0f;
+
+    private void Start()
+    {
+        Destroy(gameObject, 2.0f);
```
Comment (RyanCross) [inline Assets/Scripts/Projectile.cs]: why not use that cool variable you declared up there.

**Review by warrenscipio — APPROVED** [review]

**Review by RyanCross — APPROVED** [review]

File: Assets/Scripts/Projectile.cs (hunk 1 above)
Comment (cole-c) [reply Assets/Scripts/Projectile.cs]: Should be done

Comment (RyanCross) [reply Assets/Scripts/Projectile.cs]: frick off

**Pull request merged**